mmdet/models/dense_heads/yolox_head.py

```diff
@@ -278,16 +278,19 @@ def get_bboxes(self,
             flatten_bboxes[..., :4] /= flatten_bboxes.new_tensor(
                 scale_factors).unsqueeze(1)
 
-        result_list = []
+        dets = [], labels = []
```

Comment:
You need to separate this line into two lines without a comma

Reply:
rewrite already.